frontend/src/UnAuthenticatedSwitch.jsx

```diff
@@ -1,19 +1,16 @@
-import React, { useEffect } from "react";
+import React from "react";
 import { Routes, Route } from "react-router-dom";
 import Login from "./pages/Login";
 import Footer from "./components/Footer";
 import AlertBanner from "./components/AlertBanner";
 import UnAuthenticatedAppHeader from "./components/UnAuthenticatedAppHeader";
 import NotFound from "./pages/errorPages/NotFound";
 import Unauthorized from "./pages/errorPages/Unauthorized";
-import About from "./About";
-import EncounterSearch from "./pages/EncounterSearch";
-import Home from "./pages/Home";
 
 export default function UnAuthenticatedSwitch({ showAlert, setShowAlert }) {
   const [header, setHeader] = React.useState(true);
-  const [headerTop, setHeaderTop] = React.useState("60px");
-  const alertBannerRef = React.useRef(null);
+  // const [headerTop, setHeaderTop] = React.useState("60px");
```

Comment:
can these be removed since they're commented out?

Reply:
deleted